plugins/de.cognicrypt.staticanalyzer/pom.xml

```diff
@@ -11,6 +10,56 @@
     <version>1.0.0-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
+  
+  <properties>
+		<aetherVersion>1.1.0</aetherVersion>
+		<mavenVersion>3.3.3</mavenVersion>
+	</properties>
+  
+  <dependencies>
+		<dependency>
+			<groupId>org.eclipse.aether</groupId>
```

Comment:
Why do you need these dependencies here again? You've already defined them in the POM of core.

Reply:
You are right. These redundant dependencies have been removed.